engine/flink/kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/kafka/source/flink/KafkaSourceFactoryProcessMixin.scala

```diff
@@ -44,7 +44,7 @@ trait KafkaSourceFactoryProcessMixin
   protected def run(process: CanonicalProcess)(action: => Unit): Unit = {
     val env = flinkMiniCluster.createExecutionEnvironment()
     UnitTestsFlinkRunner.registerInEnvironmentWithModel(env, modelData)(process)
-    env.withJobRunning(process.id)(action)
+    env.withJobRunning(process.name.value)(action)
```

Comment:
Hmm, why does `MiniClusterExecutionEnvironment.withJobRunning` accept string?

Reply:
Because it is a flink Job's name, not scenario name. It is one of rare places where we should trains from our concept to someone else concept. The problem is that we very commonly use it in our tests, but it is another story that we still use teskit in a very few places. If we use testkit, this technical detail would be hidden.